Nez.Portable/Assets/Aseprite/Loader/AsepriteFileLoader.cs

```diff
@@ -90,7 +90,7 @@ public static AsepriteFile Load(string name, bool premultiplyAlpha = false)
 			}
 		}
 
-		private static AsepriteFile ReadAsepriteFile(this BinaryReader reader, string name, bool premultiplyAlpha)
+		public static AsepriteFile ReadAsepriteFile(this BinaryReader reader, string name, bool premultiplyAlpha)
```

Comment:
The issue with publically exposing this is that the method itself will dispose of the `BinaryReader` when an exception occurs.  

If you are handling the exception at the top level, but are expecting to still have the `BInaryReader` available after exception, then that would be a problem.

I would recommend a different solution (which is something I'm doing in AsepriteDotNet right now), where instead you keep this private and add an additional public method that accepts a `Stream` instead with the optional parameter of keeping the stream open after the `BinaryReader` is disposed.

Something Like this

```cs
public static AsepriteFile Load(string name, Stream stream, bool leaveOpen = false, bool premultiplyAlpha)
{
    using (BinaryReader reader = new BinaryReader(stream, System.Text.Encoding.UTF8, leaveOpen)
    {
        return AsepriteFileReader(reader, name, premultiplyAlpha);
    }
}
```

In this way, the consumer can pass the `Stream` manually if needed with the option of choosing if the stream should be closed or left open after the method finishes, whether it throws an exception or not.

Reply:
@adtennant no worries. I'm actually in the middle.of a huge rewrite of AsepriteDotNet that should be releasing soon.

It won't be to different from a consumer level to what it is right now, but I'm rewriting some internal stuff so I can make it an agnostic parser for Asepeite instead of maintaining parsing logic there and MonoGame.Aseprite, and here in Nez, and any other libs eventually.